internal/k8s/epslices/endpoint_slices.go

```diff
@@ -35,6 +35,15 @@ func IsConditionReady(conditions discovery.EndpointConditions) bool {
 	return *conditions.Ready
 }
 
+// IsConditionServing tells if the conditions represent a serving state, deferring
```

Comment:
Do we still need IsConditionReady now?

Reply:
@fedepaol yep, the documentation instructs to defer to the readiness state, if serving is nil, so I followed it literally too

```
   serving	<boolean>
     serving is identical to ready except that it is set regardless of the
     terminating state of endpoints. This condition should be set to true for a
     ready endpoint that is terminating. If nil, consumers should defer to the
     ready condition.
```

Reply:
Okay, merged the `IsConditionReady` logic into `IsConditionServing`